walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -31,6 +31,16 @@
       TimeRange afterEvent = TimeRange.fromStartEnd(event.getWhen().end(), TimeRange.END_OF_DAY, true);
       options.add(afterEvent);
     }
-    return options;
+    Collection<TimeRange> updatedOptions = new ArrayList<>();
+    for(TimeRange firstTime : options){
+      for(TimeRange secondTime : options){
+        if(firstTime.overlaps(secondTime) && !(firstTime.equals(secondTime))){
```

Comment:
Nit: the parentheses around the equality check is redundant, just use !firstTime.equals(secondTime)